### DISCUSSION TITLE
What exactly is the "copy" storage engine?

### DISCUSSION THREAD

**Discussion by jacobgraf** [lra/mackup#1937] [Q&A] [answered]
In the docs, it says when you use `engine=copy`, `Mackup will figure out where your Copy folder is and store your configuration files in it.`.

What does this even mean? What is a "Copy Folder"? Ideally, I am hoping this allows me to specify a folder to "copy" my prefs too instead of symlinking them, but the docs are so vague, I can't seem to figure out what how it's actually supposed to work.

Thanks for clarifying!

**Comment by lra** [comment, marked as answer]
It was a competitor to dropbox that died a while ago.
I removed the support for it which should avoid such confusion: https://github.com/lra/mackup/pull/1957
Thanks for raising this.
